app/controllers/bookings_controller.rb

```diff
@@ -1,2 +1,41 @@
 class BookingsController < ApplicationController
+  before_action :find_dress
+
+  def new
+    @booking = Booking.new
+  end
+
+  def create
+    @booking = Booking.new(booking_params)
+    @booking.dress = @dress
+    @booking.user = @current_user
```

Comment:
Does current_user need the @ mark?

Reply:
I don't think it takes an @
I am not sure actually, but, does the @booking.user and dress need to be set this way if they are included in the booking_params? If we adjust the booking_params then we might be OK just doing Booking,new(booking_params) :)

Reply:
Current user doesnt need the @ symbol, you're right.

Reply:
I’ll change that too 👍

Reply:
@Gazwai Could you give what @CamillieFu is saying a try pls? Try deleting @booking.dress = @dress and @booking.user = @current_user and see if we can create a booking. I think it should work too but not entirely sure.

Reply:
Once we add pundit it seems like we will need to do it the way that Gary has it? Might be better leaving it as is/chatting with Yann tonight - try downloading Pundit as well.

<img width="429" alt="Screen Shot 2022-01-25 at 15 03 08" src="https://user-images.githubusercontent.com/86721807/150920212-a874ed4c-41b7-4388-9a95-892e62b1db12.png">
t